### ISSUE TITLE
[Suggestion] List view for books

### ISSUE THREAD

**Issue by jaydentanke** [diderot-edu/diderot-feedback#97]
It is currently a little difficult to find the right book when a course has many books with many chapters. Since the books are displayed in a n x 3 grid, it is necessary to scroll down while scanning left and right to find the correct one.

It would be great if we had a list view of books, with tooltips to expand/hide the chapters of a book.

**Comment by umutacar** [comment]
yes, agreed.  it would be good to provide random access to books.  in the interim, note that your "Chapters" list caches the chapters and books that you have accessed and it will remember every book.  You can jump to a book by using this list.  Here is how mine looks like for example.  I use this as a shortcut to books, in a way similar to the list that you have suggest.
![image](https://user-images.githubusercontent.com/6068765/110879385-e0a6a180-82aa-11eb-8e0b-3832d63f6260.png)
